plugins/polio/migrations/0145_alter_vaccineauthorization_expiration_date.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.15 on 2023-09-13 09:13
+
+from django.db import migrations, models
+import datetime
+
+
+# handle old records with a None date.
+def populate_expiration_date(apps, schema_editor):
+    VaccineAuthorization = apps.get_model("polio", "VaccineAuthorization")
+
+    VaccineAuthorization.objects.filter(expiration_date=None).update(expiration_date=datetime.datetime.now())
```

Comment:
I'm not sure about this. it will lead to incorrect data. Doesn't really matter if there's no data in prod though

Reply:
It will only run at migration if there is no date which is impossible now as we dont allow it. Without this, I don't see how to rung the migrations on tests environments. It can be removed later tho